lae_automation/test/strategies.py

```diff
@@ -342,7 +342,7 @@ def urls():
     )
 
 
-def deployment_configuration():
+def deployment_configurations():
```

Comment:
Just trying to shift the names of everything in this module towards the Hypothesis convention of plurals for strategies.